app/views/pages/_editor_guide_text.html.erb

```diff
@@ -225,7 +225,7 @@
     <p>All you need is the data-id code from the embed link:</p>
     <pre>
       <span style="color: gray;"># Given this embed link:</span>
-      < script async class="speakerdeck-embed"
+      <script async class="speakerdeck-embed"
```

Comment:
Hmm I think this was giving an issue that it would render the script instead of showing the code.

Reply:
Thanks for pointing that out! Looks like Ben wanted to make some improvements to it, so I'm closing this PR.